parser-typechecker/src/Unison/Codebase/Path.hs

```diff
@@ -399,10 +413,6 @@ instance Convert Path [NameSegment] where convert = toList
 
 instance Convert HQSplit (HQ'.HashQualified Path) where convert = unsplitHQ
 
-instance Convert Path Name where convert = toName
-
-instance Convert Path' Name where convert = toName'
-
```

Comment:
These can become `Parse` instances instead, if that's useful.

Reply:
I don't believe we ever need to use it like that right now, and generally I find using the explicit combinators more clear 😄